kotlin/perform-valve-isolation-trace/src/main/java/com/esri/arcgisruntime/sample/performvalveisolationtrace/MainActivity.kt

```diff
@@ -249,30 +434,29 @@ class MainActivity : AppCompatActivity() {
      */
     private fun performTrace(
         utilityNetwork: UtilityNetwork,
-        traceConfiguration: UtilityTraceConfiguration,
-        startingLocation: UtilityElement
+        traceConfiguration: UtilityTraceConfiguration
```

Comment:
Remove line 433. `@param startingLocation` not needed since it's not a function argument.